quality/transform_load_dlt.py

```diff
@@ -144,3 +144,20 @@ def quality_judiciary_silver():
         )
         .filter(F.col('amount').isNotNull())
     )
+
+# COMMAND ----------
+
+
+@dlt.table(name=f'quality_total_subnat_silver')
+def quality_total_silver():
+    bronze = dlt.read('quality_cci_bronze')
+    year_cols = list(col_name for col_name in bronze.columns if col_name.isnumeric())
+    return (bronze
+        .filter(F.col('category_code') == 'EXP_ECON_SBN_TOT_SPE_EXE')
+        .melt(ids=["country_name", "approved_or_executed"], 
+            values=year_cols, 
+            variableColumnName="year", 
+            valueColumnName="amount"
+        )
+        .filter(F.col('amount').isNotNull())
+    )
```

Comment:
I merged your changes in PR #4 so having the same change here may cause merge conflict. I suggest doing a "pull" in databricks workspace (you might have to do it twice, once on the main branch and again on this working branch) and see if it prompts you on merge conflict. Let me know if you need help working through that.